brave/brave-core
# fix(wallet): Send NFT Modal Title

## Description 
Updated the `Send NFT` modal to say `Select an NFT to send`


Resolves <https://github.com/brave/brave-browser/issues/27374>

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Go to the `Send` tab and switch to `Send NFT`.
2. Click `Select NFT`.
3. The modal title should be `Select an NFT to send`, not `Select a token to send`.

https://user-images.githubusercontent.com/40611140/207649936-1bf594ff-24e7-42a4-a79e-aedd40bcabe1.mov
